Source/cmake/OptionsWPE.cmake

```diff
@@ -157,9 +157,10 @@ if (USE_SOUP2)
     set(SOUP_API_VERSION 2.4)
     set(WPE_API_VERSION 1.0)
 else ()
+    SET_AND_EXPOSE_TO_BUILD(ENABLE_2022_GLIB_API ON)
```

Comment:
I guess this is to have a common ifdef for new api common to gtk and wpe. Adding an unused variable in this patch looks weird to me in any cae, I would add it in the first patch actually using it.

Reply:
Yes, we discussed this in https://bugs.webkit.org/show_bug.cgi?id=242409 and several times in our private chat. At least, sort of. The discussion felt rather one-way. :P

I wanted to do this in a separate PR so that we can agree on and land the new build flag distinct from any particular API changes.